### PR TITLE
Add nvargus_nvraw into Kirkstone l4t r32.7.x

### PR DESCRIPTION
**Pull request by paroque28**
Add nvargis_nvraw utility, useful for debugging, getting raw data and listing sensors:

```bash
$ nvargus_nvraw --lps
nvargus_nvraw version 1.11.0
Number of supported sensor entries 5
Entry  Source Mode      Uniquename             Resolution   FR  BitDepth  Mode
Index  Index  Index                                             CSI Dyn   Type
  0      0      0         imx334_front_framos   3864x2180   59  10  10    Bayer       
  1      0      1         imx334_front_framos   1920x1080  120  10  10    Bayer       
  2      0      2         imx334_front_framos   1944x1080   59  12  12    Bayer       
  3      0      3         imx334_front_framos   3864x2180   59  12  12    Bayer       
  4      0      4         imx334_front_framos   1920x1080  120  12  12    Bayer    
  ```

### CODE REVIEW

**Comment by paroque28** [comment]
Is there any reason why we don't add this tool?

**Comment by ichergui** [comment]
> Is there any reason why we don't add this tool?

There is no reason thought.
Thanks for this PR. 
LGTM

**Review by ichergui — CHANGES_REQUESTED** [review]
You need to add `{PN}-nvargus-nvraw` in `PACKAGES` list.

File: recipes-bsp/tegra-binaries/tegra-libraries-camera_32.7.5.bb [hunk 1]
```diff
@@ -71,6 +72,7 @@ FILES_SOLIBSDEV = ""
 SOLIBS = ".so*"
 FILES:${PN} += "${libdir}/libv4l/plugins"
 FILES:tegra-libraries-argus-daemon-base = "${sbindir}/nvargus-daemon"
+FILES:{PN}-nvargus-nvraw = "${sbindir}/nvargus_nvraw"
```
Comment (ichergui) [inline recipes-bsp/tegra-binaries/tegra-libraries-camera_32.7.5.bb:75]: You need to add {PN}-nvargus-nvraw in PACKAGES list.

**Review by ichergui — CHANGES_REQUESTED** [review]
Please check my comment.

**Review by ichergui — APPROVED** [review]
LGTM

**Comment by paroque28** [comment]
Thank you for your comments @ichergui

**Comment by paroque28** [comment]
Ok @ichergui sorry for the change, I grabbed the changes from the wrong place and it contained those errors, missing package and missing `$` in `${PN}`.

I confirmed that the changes here compile and I tested on a device.

**Pull request merged**